ulib/Makefile.verify

```diff
@@ -34,3 +34,34 @@ LowStar.Printf.fst.checked: USE_EXTRACTED_INTERFACES=
 verify-core: $(addsuffix .checked, $(filter-out $(EXTRA) ,$(FSTAR_FILES)))
 
 verify-extra: $(addsuffix .checked, $(EXTRA))
+
+# Benchmarking rules
+#
+# we want to run FStar with just the target file as that being checked
+# ideally all the .checked files will exist and we will:
+#  - move them to the side
+#  - execute the benchmark
+#  - move the checked file back
+#
+
+# a.fst.bench will move the a.fst.checked binary and then move it back
```

Comment:
should this target be .PHONY? it may be that the .fst has not changed, but you still want to re-run the benchmark

Reply:
I guess it's ok because it's a pseudo-target and not an actual file...

Reply:
You are right, rerunning the benchmarks will not regenerate without cleaning the .bench files. I've a fix for this.